src/main/com/aca/homework/week10/text/reader/TextReader.java

```diff
@@ -0,0 +1,29 @@
+package com.aca.homework.week10.text.reader;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class TextReader {
+
+    public static List<String> readLines() throws FileNotFoundException {
+        List<String> lines = new ArrayList<>(1000);
```

Comment:
the linked list suits better in this problem; also, you do not know the exact number of lines